packages/grafana-schema/src/raw/composable/prometheus/dataquery/x/PrometheusDataQuery_types.gen.ts

```diff
@@ -30,7 +30,7 @@ export interface PrometheusDataQuery extends common.DataQuery {
   /**
    * The actual expression/query that will be evaluated by Prometheus
    */
-  expr: string;
+  expr?: string;
```

Comment:
@itsmylife this is the only change I made, but somehow the output no longer includes RefID, QueryType, and Hide? 
I'm probably doing something silly?

Reply:
This is a generated file. You cannot edit this directly. You have to edit the `.cue` file for this. 
Also, I am not sure if that's what we want in that case. If `expr` can be null and I see you handled it everywhere with `??`, so it cannot be null.

Reply:
I think you did change the `prometheus/datasource.cue` file and run `make gen-cue`. Is that right?